content/en/data_security/real_user_monitoring.md

```diff
@@ -0,0 +1,125 @@
+## Compliance frameworks
+RUM can be configured for compliance with many standards and regulatory frameworks, including, but not limited to:
+
+- GDPR
+- HIPAA 
+- ISO
+- CCPA/CPRA
+
+## Privacy restrictions
+By default, there are some privacy restrictions in place that protect user data to help comply with regulatory and standards frameworks.
+
+### Browser RUM use of cookies
+Browser RUM requires **first party cookies** to be enabled on an end user's browser to collect data. If required by the jurisdictions in which you operate, your pages should be configured so that the end user must accept cookies RUM is initialized and begins collecting data.
+
+### Mobile RUM consent management
+Mobile RUM tracking is only run upon user consent. If the end user accepts the RUM tracking, we track their activity and session experience. If the user declines the RUM tracking, we do not track their activity and session experience.
+
+## Privacy options
+You have several options and tools when it comes to collecting and redacting data captured by RUM.
+
+### Event tracking
+An [event][14] is a user interaction with specific elements of your site or app. Events can be automatically captured via the SDK or sent via custom actions. You can turn off automatic tracking of user interactions and page views to only capture the interaction of your choice. By default, RUM uses target content to generate action names from actions automatically collected by the SDK. You can [explicitly override][5] this behavior with any given name.
+
+The data we track automatically contains primarily technical information, much of which doesn't include personal identifying information. Data that is captured by RUM can be further redacted before it is sent and stored in Datadog through advanced configuration options for the following methods:
+
+- [beforeSend API][1]
+- [iOS][2]
+- [Android][3]
+- [Flutter][4]
+
+### Transmit RUM events through a proxy server
+You can transmit all RUM events through your own [proxy server][15] so that end user devices never directly communicate with Datadog.
+
+### User identity tracking
+By default, there is **no tracking of users' identity**. Each session has a unique `session.id` tied to it, which anonymizes the data, but allows you to understand trends. You have the option of writing code to capture [user data][6] such as name and email address, then using them to [enrich and modify][7] RUM sessions, but they are not required.
+
+### Data retention
+After you have configured the event capture, events are stored in Datadog. You can decide how long your captured events and properties stay in Datadog.
+
+By default, data retention for production environments is:
+
+- 30 days for sessions, views, actions, errors, and session recordings
+- 15 days for resources and long tasks
```

Comment:
Small suggestion to consider adding periods to the end of each bullet point since, when combined with the lead-in text, the bullet points form a sentence. But honestly, I could go either way.